uaclient/config.py

```diff
@@ -4,7 +4,7 @@
 import logging
 import os
 import yaml
-from collections import namedtuple
+from collections import namedtuple, OrderedDict
```

Comment:
I assume it's ordered dict so that the help output comes out in the right order. It might be worth highlighting that at the right place so that doesn't get changed around and is a documented decision.

Reply:
Yes, that's exactly why I am using it. I will add comment for that reasoning